src/Criteo.OpenApi.Comparator.UTest/Resource/issue_49_false_1015_equal_schemas/diff.json

```diff
@@ -0,0 +1,13 @@
+[
```

Comment:
This test shows that when two parameters are present with the same name, #49 does not occur. This is the equal case, where the schemas do not differ.